lib/src/upgrade_alert.dart

```diff
@@ -6,6 +6,8 @@ import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
 import 'package:http/http.dart' as http;
 
+import '../upgrader.dart';
+import 'upgrader.dart';
```

Comment:
@PonnamKarthik Thanks for the PR. The travis-ci build of this branch is failing, in one case for this duplicate import. Perhaps this is due to a merge problem?
https://travis-ci.org/larryaasen/upgrader/pull_requests

Reply:
@larryaasen updated the Code